llvm/lib/Target/DirectX/DXILShaderFlags.h

```diff
@@ -60,21 +62,54 @@ struct ComputedShaderFlags {
     return FeatureFlags;
   }
 
-  static ComputedShaderFlags computeFlags(Module &M);
+  uint64_t getModuleFlags() const {
+    uint64_t ModuleFlags = 0;
+#define DXIL_MODULE_FLAG(DxilModuleBit, FlagName, Str)                         \
+  ModuleFlags |= FlagName ? getMask(DxilModuleBit) : 0ull;
```

Comment:
> We have this exact expansion inside the `operator uint64_t` definition above. Should we maybe refactor this so that one uses the other instead of duplicating the expansion?

Refactored.